src/main/cpp/RobotContainer.cpp

```diff
@@ -194,130 +157,61 @@ void RobotContainer::ConfigureButtonBindings()
-  // frc2::JoystickButton(&m_driverController, 4).ToggleOnTrue(AprilTagAim(m_limelight, m_drive, m_driverController, m_shooter).ToPtr());
-
-  //TODO adjust deadzone so the robot will not be at an angle when aiming, also implement this fucntion into the state machine
-  // frc2::JoystickButton(&m_driverController, 6).ToggleOnTrue(AutoAprilTag(m_limelight, m_drive).ToPtr());
-
-  // //start PICKUP state
-  // //frc2::JoystickButton(&m_driverController, 5).ToggleOnTrue(m_intakeShoot.Pickup());
-  // //frc2::JoystickButton(&m_driverController, 5).ToggleOnFalse(m_intakeShoot.PickupStop());
-
 }
 
-float RobotContainer::DeadzoneCubed(float x){
+float RobotContainer::DeadzoneCubed(float x)
+{
   x = x * x * x;  // exponetial curve, slow acceleration at begining
 
-  if ((x < 0.001) &&  (x > -0.001)){
-    x=0;
+  if ((x < 0.001) &&  (x > -0.001))
+  {
+    x = 0;
   }
-  else if (x >= 0.001){
-    x = x ;//- 0.1;
+  else if (x >= 0.001)
+  {
+    x = x ;
   }
-  else if (x <= -0.001){
-    x = x ;//+ 0.1;
+  else if (x <= -0.001)
+  {
+    x = x ;
   }
 
   return(x);
- 
 }
 
-float RobotContainer::Deadzone(float x){
-  if ((x < 0.1) &&  (x > -0.1)){
-    x=0;
+float RobotContainer::Deadzone(float x)
+{
+  if ((x < 0.1) && (x > -0.1))
+  {
+    x = 0;
   }
-  else if (x >= 0.1){
+  else if (x >= 0.1)
+  {
     x = x - 0.1;
   }
-  else if (x <= -0.1){
+  else if (x <= -0.1)
+  {
     x = x + 0.1;
   }
 
   return(x);
 }
 
-// frc2::CommandPtr RobotContainer::GetAuxilaryStateMachine()
-// {
-//   return StateMachine(
-//       m_drive,
-//       m_limelight,
-//       m_arm, 
-//       m_climb,
-//       m_color, //can remove?
-//       m_intake,
-//       m_shooter,
-//       m_driverController,
-//       m_auxController
-//     ).ToPtr();
-// }
-
-/*
-frc2::CommandPtr RobotContainer::GetDriveStateMachine(){
-  return DriveStateMachine(
-    m_drive,
-    m_limelight, 
-    m_driverController, 
-    m_auxController, 
-    driveShooterMessager
-  ).ToPtr();
-}
-*/
-
-// frc2::CommandPtr RobotContainer::GetAutoAuxilaryStateMachine(){
-//   return AutoAuxilaryStateMachine(
-//       m_arm, 
-//       m_climb, //can remove?
-//       m_color, //can remove.
-//       m_intake,
-//       m_shooter, 
-//       m_driverController,
-//       m_auxController,
-//       driveShooterMessager
-//     ).ToPtr();
-// }
-
-// frc2::CommandPtr RobotContainer::GetAutoDriveStateMachine(){
-
-//   // m_drive.ResetOdometry(B_1Waypoints[0]);
-
-//   return AutoDriveStateMachine(
-//     m_drive,
-//     m_limelight,
-//     m_driverController,
-//     m_auxController,
-//     driveShooterMessager,
-//     path
-//   ).ToPtr();
-// }
-
-// void RobotContainer::SetRanAuto(bool ranAuto){
-//   m_drive.SetRanAuto(ranAuto);
-// }
-
 frc2::CommandPtr RobotContainer::GetAutonomousCommand()
 {
   m_shooter.zeroIntergralVal();
 
   chosenAuto = m_chooser.GetSelected();
 
-
-  std::vector<frc::Pose2d> twoNotePoses{
+  std::vector<frc::Pose2d> twoNotePoses
+  {
```

Comment:
Some of these paths in the .cpp could probably be removed or moved to the .h

Reply:
we're going to resolve this later